src/pybind/mgr/volumes/pathspec.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
I prefer that this module be at pybind/mgr/volumes/fs/ folder and named something else, e.g., subvolspec.py